dist/simulator.js

```diff
@@ -1570,6 +1569,23 @@ function debug_name(card, hideStats) {
     return output;
 }
 
+function debug_find_skill(target, s) {
```

Comment:
💡 How about we update `debug_skill` to take in `target` and then include enhancement when it calculates `x`.

That would keep the logs for `imbue` and `enhance` consistent.